engine/api/workflow/dao_node_job_run_info.go

```diff
@@ -0,0 +1,60 @@
+package workflow
+
+import (
+	"database/sql"
+	"encoding/json"
+	"time"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
+)
+
+//loadNodeRunJobInfo load infos (workflow_node_run_job_infos) for a job (workflow_node_run_job)
+func loadNodeRunJobInfo(db gorp.SqlExecutor, jobID int64) ([]sdk.SpawnInfo, error) {
+	query := "SELECT spawninfos FROM workflow_node_run_job_info WHERE workflow_node_run_job_id = $1"
```

Comment:
suggestion:
to avoid scan and scrap the resultset you could do (omitting err management)
````
res := []struct {
      Bytes sql.NullString `db: spawninfos`
}{}
db.Select(&res, query, jobID)
spawnInfos := make(sdk.SpawnInfo, len(res))
for i, r := range res {
    gorpmapping. JSONNullString(r.Bytes, &spawnInfos[i])
}
`````

Reply:
yes, I'll test that
(workflow_node_run_job_info contains a slice of sdk.SpawnInfos. So, for a job, we had to concat rows result to returns the whole spawninfos)